local/server/server.go

```diff
@@ -34,3 +37,34 @@ func Serve(s *Server, scootdir string) error {
 func (s *Server) Echo(ctx context.Context, req *protocol.EchoRequest) (*protocol.EchoReply, error) {
 	return &protocol.EchoReply{Pong: "Pong: " + req.Ping}, nil
 }
+
+func (s *Server) Run(ctx context.Context, req *protocol.Command) (*protocol.ProcessStatus, error) {
+	cmd := &runner.Command{req.Argv, req.Env,
+		time.Duration(req.Timeout.Seconds)*time.Second + time.Duration(req.Timeout.Nanos),
+	}
+	status, err := s.runner.Run(cmd)
+	if err != nil {
+		return nil, err
+	}
+	return runnerToProtocolStatus(status), nil
+}
+
+func (s *Server) Status(ctx context.Context, req *protocol.StatusQuery) (*protocol.ProcessStatus, error) {
+	status, err := s.runner.Status(runner.RunId(req.RunId))
+	if err != nil {
+		return nil, err
+	}
+
+	return runnerToProtocolStatus(status), nil
+}
+
+func runnerToProtocolStatus(status *runner.ProcessStatus) *protocol.ProcessStatus {
```

Comment:
I'm not sure this belongs here. We don't want protocol to leak into the runner code, so it can't go there, but maybe somewhere in the protocol dir?